doc/conf.py

```diff
@@ -5,13 +5,11 @@
 # dir.
 #
 # The contents of this file are pickled, so don't put values in the namespace
-# that aren't picklable (module imports are okay, they're removed
+# that isn't pickable (module imports are okay, they're removed
 # automatically).
 #
 # All configuration values have a default value; values that are commented out
 # serve to show the default value.
-
-import logging
```

Comment:
The `logging` module is used below (see flake8) comment.  So we can not remove it.